openshift/ftest-standalone/src/test/resources/arquillian.xml

```diff
@@ -6,6 +6,15 @@
 
   <extension qualifier="openshift">
     <property name="env.config.resource.name">hello-openshift.json</property>
+
+    <!-- configuring remote cluster -->
+
+    <!--
+    <property name="namespace.use.existing">existing-namespace</property>
+    <property name="kubernetes.master">https://api.starter-us-east-2.openshift.com</property>
+    <property name="cube.auth.token">token</property>
```

Comment:
we don't need token and username/password at the same time, maybe OR will help in between for user to understand if we are going to keep this comment